k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash

```diff
@@ -7,14 +7,18 @@ MONGODB_BACKUP_CONF_FILE=/etc/mongodb-mms/backup-agent.config
 mms_api_keyfile_path=`printenv MMS_API_KEYFILE_PATH`
 mms_groupid_keyfile_path=`printenv MMS_GROUPID_KEYFILE_PATH`
 ca_crt_path=`printenv CA_CRT_PATH`
-backup_crt_path=`printenv BACKUP_PEM_PATH`
+backup_pem_path=`printenv BACKUP_PEM_PATH`
 
-if [[ -z "${mms_api_keyfile_path}" || \
-    -z "${ca_crt_path}" || \
-    -z "${backup_crt_path}" || \
-    -z "${mms_groupid_keyfile_path}" ]]; then
-  echo "Invalid environment settings detected. Exiting!"
+if [[ -z "${mms_api_keyfile_path:?MMS API Keyfile path not specified. Exiting!}" || \
```

Comment:
Can we use the actual env var in the error message?
`MMS API Keyfile path not specified.` => `MMS_API_KEYFILE_PATH not specified.`

Same for every other line in the PR?